### DISCUSSION TITLE
Bug Report: Two AMMUNITION_OUT connectors appear on empty weapon loaders

### DISCUSSION THREAD

**Discussion by UserC2** [FakeFishGames/Barotrauma#14329] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Two AMMUNITION_OUT connectors appear on empty weapon loaders.
Once a weapon is purchased for that loader, the second AMMUNITION_OUT connector disappears.

### Reproduction steps

1. Get into a submarine with an empty turret hardpoint.
1. Use a screwdriver to modify the wiring on the empty hardpoint.
1. Observe two AMMUNITION_OUT connectors.
1. Purchase a weapon for that hardpoint.
1. Use a screwdriver to modify the wiring on the hardpoint again.
2. Observe one AMMUNITION_OUT connector.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/d1d787aaedba1d2a71392c57b49ea94ec7834867 (commit in our private repo). Fix coming up in a future update!
